tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -18,15 +23,17 @@ describe(__filename, () => {
   const fakeAuthorTwo = { ...fakeAuthor, username: 'test2', id: 61 };
   const fakeAuthorThree = { ...fakeAuthor, username: 'test3', id: 71 };
 
-  function fakeAddons() {
+  function fakeAddons(type = ADDON_TYPE_EXTENSION) {
```

Comment:
We usually use an object to gather all the arguments of a function, because it is more convenient if we want to add new arguments (and also because we don't have to remember the order of the args).

```js
function fakeAddons({ type = ADDON_TYPE_EXTENSION } = {}) {
  // ...
}
```